packages/react-devtools-shared/src/utils.js

```diff
@@ -653,6 +657,9 @@ export function formatDataForPreview(
       } else {
         return `${name}(${data.size})`;
       }
+    case 'opaque_iterator': {
+      return `${data.constructor.name}(${data.size})`;
```

Comment:
I think this will result in the string `"(undefined)"` (at least based on my testing, using a similar approach as mentioned in #19726).

Reply:
Maybe just `String(data) `can work for all

Reply:
That would result in a different format than we're looking for here.

Reply:
`${data.constructor.name}(${data.size})` I think we cannot have this format, because we cannot have the `size` of  generator

Reply:
Yeah, I know. That's what I commented above :)